src/data/structure/Recommendation.java

```diff
@@ -0,0 +1,33 @@
+package data.structure;
+
+/**
+ * Represents a recommendation based on a UserProfile with its associated Preferences
+ *    (i.e. the user's preferences).  Contains the Preference that is most correlated
+ *      with the user's preferences as well as total correlation score.
+ * @author benjaminposerow
+ *
+ */
+public class Recommendation {
```

Comment:
I started making myself do this proactively after Anthony spoke once about why he creates those for any POJO. It's really just a good practice, as it's almost inevitable that POJOs will eventually be used in HashMaps or other Collections where they become necessary.